orc8r/lib/go/registry/registry.go

```diff
@@ -206,12 +207,32 @@ func (r *ServiceRegistry) GetAnnotation(service, annotationName string) (string,
 	if !ok {
 		return "", fmt.Errorf("service %s not registered", service)
 	}
-	labelValue, ok := location.Annotations[annotationName]
+	annotationValue, ok := location.Annotations[annotationName]
 	if !ok {
 		return "", fmt.Errorf("service %s doesn't have annotation values for %s", service, annotationName)
 	}
 
-	return labelValue, nil
+	return annotationValue, nil
+}
+
+// GetAnnotationFields returns the split fields of the value for the passed
+// annotation name.
+// First splits by field separator, then strips all whitespace
+// (including newlines). Empty fields are removed.
+func (r *ServiceRegistry) GetAnnotationFields(service, annotationName, fieldSeparator string) ([]string, error) {
```

Comment:
`GetAnnotationListValue` or `GetListAnnotationValue` or something like that. This name makes me think you're grabbing all the annotation keys on a service. I think you can probably just use the const for the separator too